TheFourFifths/consus
# Functional test: Adding a new model

### Summary

This is the first functional test for the server. It uses the API to test adding a new model.

### Checklist


### How to Review

1. Check out `new-model-test-TFF-137`
2. `npm install` and `npm test`
3. Verify that the functional tests are running and passing
4. Review the tests for thoroughness and clarity

### Links

- [TFF-132](https://msoese.atlassian.net/browse/TFF-132)
